Assets/MixedRealityToolkit.Providers/WindowsMixedReality/Shared/DotNetAdapter/DotNetAdapter.csproj

```diff
@@ -1,26 +0,0 @@
-﻿<Project>
```

Comment:
Looks like this file didn't get re-committed (will need to `git add -f` it, since `.csproj` is in the gitignore)

Reply:
good catch. thanks!

Reply:
done